cmd/root.go

```
@@ -23,12 +26,26 @@ var Root = New("ocip", "start registry service")
 	)
 	return root
 }
+
+func newCmdVersion() *cobra.Command {
+	return &cobra.Command{
+		Use:   "version",
+		Short: "Print the version",
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, _ []string) {
+			// cmd.Println falls back to stderr; scripts pipe this output.
+			fmt.Fprintln(cmd.OutOrStdout(), version.Version)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)